portfolio/src/main/webapp/script.js

```diff
@@ -46,6 +47,26 @@ function createListElement(text) {
   return liElement;
 }
 
+/** Checks if the user has inputted any text upon clicking submit. */
+function isInputEmpty() {
+  const comment = document.getElementById('text-input').value;
+  if (!comment || comment === '' || isInputOnlySpaces(comment)) {
+    alert('Please enter a valid comment.');
+    return false;
+  }
+  return true;
+}
+
+/** Checks if the user has inputted only spaces. */
+function isInputOnlySpaces(input) {
```

Comment:
prefer to use a regex pattern, then you can check for spaces, new lines, tabs... (or if it contains at least one alphaneumeric character)